src/cpp/rserver-dev.in

```diff
@@ -35,5 +35,5 @@ trap cleanup SIGINT
 RS_CRASH_HANDLER_PATH="$(pwd)/server/crash-handler-proxy/crash-handler-proxy" \
    RS_CRASHPAD_HANDLER_PATH="${RSTUDIO_TOOLS_ROOT}/crashpad/crashpad/out/Default/crashpad_handler" \
    RS_DB_MIGRATIONS_PATH="${CMAKE_CURRENT_SOURCE_DIR}/server/db" \
-   server/rserver --server-user $(whoami) --config-file conf/rserver-dev.conf "$@"
+   server/rserver --server-user $(whoami) --auth-none=1 --config-file conf/rserver-dev.conf "$@"
```

Comment:
I feel like we can skip the authentication step when working in open source development builds now.

Reply:
Seems reasonable.